app/src/main/java/com/techyourchance/coroutines/exercises/homework/useCase/EmulateNetworkCallUseCase.kt

```diff
@@ -0,0 +1,15 @@
+package com.techyourchance.coroutines.exercises.homework.useCase
+
+import kotlinx.coroutines.CoroutineName
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.delay
+import kotlinx.coroutines.withContext
+
+class EmulateNetworkCallUseCase {
+
+    suspend fun emulateNetworkCall() : List<String> =
+        withContext(CoroutineName("use case network service") + Dispatchers.IO) {
```

Comment:
Almost same comment about hardcoded values: It is always useful to store Numbers and Strings (even lists of strings) on named values or variables